client/components/thank-you-v2/index.tsx

```diff
@@ -0,0 +1,36 @@
+import { ConfettiAnimation } from '@automattic/components';
+import { TranslateResult } from 'i18n-calypso';
+import ThankYouFooter, { ThankYouDetailProps } from './footer';
+import ThankYouHeader from './header';
+import ThankYouUpsell, { ThankYouUpsellProps } from './upsell';
+
+import './style.scss';
+
+interface ThankYouV2Props {
+	title: string | TranslateResult;
+	subtitle: string | TranslateResult;
+	headerButtons?: React.ReactNode;
+	products?: React.ReactNode;
+	footerDetails?: ThankYouDetailProps[];
+	upsellProps?: ThankYouUpsellProps;
+}
+
+const ThankYouV2: React.FC< ThankYouV2Props > = ( props: ThankYouV2Props ) => {
+	const { title, subtitle, headerButtons, products, footerDetails, upsellProps } = props;
+
+	return (
+		<div className="thank-you__container">
+			<ConfettiAnimation delay={ 1000 } />
```

Comment:
More of an FYI: p58i-gah-p2

> Audit and trim down the confetti congrats in Calypso. Way too much confetti across the product.

Reply:
Ah, good to know! I've removed it. It was kinda fun for me to see the animation during development 😆

Reply:
Cool! Looks like the animation was never part of the Domain Only Congrats page, so we should be good to remove it for this PR.